src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/add_product_row.html.twig

```diff
@@ -28,7 +28,7 @@
     {{ form_row(addProductRowForm.product_id) }}
     {{ form_row(addProductRowForm.tax_rate) }}
     {{ form_label(addProductRowForm.search) }}
-    {{ form_widget(addProductRowForm.search) }}
+    {{ form_widget(addProductRowForm.search, { 'attr': {'data-currency': orderForViewing.currencyId }}) }}
```

Comment:
Hi @sowbiba 

Thank you ... !

You should inject this data in the controller when creating the form rather than in the template (`in PrestaShopBundle\Controller\Admin\Sell\Order::viewAction`)
See : https://github.com/PrestaShop/PrestaShop/blob/1.7.7.x/src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php#L412

We talked about it with @jolelievre  a few weeks ago, I don't know if this case is concerned. The idea is to keep consistency.

WDYF @jolelievre  ?

Reply:
I actually made the same feedback in my review ^^

Reply:
Fixed. Thanks @PululuK and @jolelievre